meta.json

```diff
@@ -1,26 +1,16 @@
 {
```

Comment:
Now #979 has been merged and the npm users have been removed, we can just delete this file now 🎉

Reply:
done!